modules/publicize/publicize.php

```diff
@@ -418,6 +419,91 @@ function save_meta( $post_id, $post ) {
 		// Next up will be ::publicize_post()
 	}
 
+	public function update_published_message( $messages ) {
+		global $post_type, $post_type_object, $post;
+		if ( ! $this->post_type_is_publicizeable( $post_type ) ) {
+			return $messages;
+		}
+		$view_post_link_html = '';
+		$viewable = is_post_type_viewable( $post_type_object );
+		if ( $viewable ) {
+			$view_text = __( 'View post' );
```

Comment:
Needs a textdomain.

Reply:
No I am intentionally sing the core text domain here. But that should be explained in a comment. :)

Reply:
👍 I actually wondered that right after I commented :)